modules/config-fields-clair-auth.adoc

```diff
@@ -16,3 +15,19 @@ The following authorization configuration fields are available for Clair.
 
 | **.psk.iss** | String | A list of JWT issuers to verify. An empty list accepts any issuer in a JWT claim.
 |===
+
+[discrete]
+== Example authorization configuration
+
+The following `authorization` snippet is for a minimal configuration. 
+
+.Example authorization configuration
+[source,yaml]
+----
+# ...
+auth:
+  psk:
+    key: ""
+    iss: []
+# ...
+----
```

Comment:
This would result in no clients being able to connect because nothing is listed in `iss`. I think the upstream documentation is also incorrect on this point.